ui/pages/extensions.js

```diff
@@ -201,11 +211,16 @@ const Extensions = ({ classes, toggleCatalogContent,  enqueueSnackbar, closeSnac
 }
 
 const mapStateToProps = state => ({
-  catalogVisibility : state.get('catalogVisibility')
+  catalogVisibility : state.get('catalogVisibility'),
+  capabilitiesRegistry : state.get('capabilitiesRegistry')
 })
 
 const mapDispatchToProps = dispatch => ({
   toggleCatalogContent : bindActionCreators(toggleCatalogContent, dispatch)
 })
 
+export const extensions = {
+  meshmap : "MeshMap"
```

Comment:
```suggestion
  MESHMAP : "meshmap"
```
So that you don't need to do `.toLowerCase()`

Reply:
Noted! I will do that.